scripts/deploy.sh

```diff
@@ -32,12 +32,12 @@ if [[ "$TRAVIS_BRANCH" == "develop" ]]; then
 
     echo "docker push sqlflow/sqlflow:$DOCKER_TAG"
     echo "$DOCKER_PASSWORD" | docker login --username "$DOCKER_USERNAME" --password-stdin
-    docker build -t sqlflow/sqlflow:$DOCKER_TAG -f ./Dockerfile .
+    docker tag sqlflow:latest sqlflow/sqlflow:$DOCKER_TAG
```

Comment:
I believe we need more comments here explaining why we don't need to build before tagging.